src/lib/visualization/pqp_visualizer.cpp

```diff
@@ -7,6 +7,7 @@
 #include "operators/limit.hpp"
 #include "operators/projection.hpp"
 #include "operators/table_scan.hpp"
+#include "utils/format_bytes.hpp"
```

Comment:
?

Reply:
Removed it in another file which did not use any byte formatting. And the PQP visualizer uses format_bytes.